Tasks/NuGet/nuget.ts

```diff
@@ -0,0 +1,94 @@
+import * as ngToolRunner from "nuget-task-common/NuGetToolRunner";
+import * as nutil from "nuget-task-common/Utility";
+import * as tl from "vsts-task-lib/task";
+global['_vsts_task_lib_loaded'] = true;
+import * as path from "path";
+import * as auth from "nuget-task-common/Authentication";
+
+import locationHelpers = require("nuget-task-common/LocationHelpers");
+import nuGetGetter = require("nuget-task-common/NuGetToolGetter");
+import peParser = require('nuget-task-common/pe-parser/index');
+
+class NuGetExecutionOptions {
+    constructor(
+        public nuGetPath: string,
+        public environment: ngToolRunner.NuGetEnvironmentSettings,
+        public command: string,
+        public args: string
+    ) { }
+}
+
+async function main(): Promise<void> {
+    let buildIdentityDisplayName: string = null;
+    let buildIdentityAccount: string = null;
+
+    try {
+        tl.setResourcePath(path.join(__dirname, "task.json"));
+        let versionSpec: string = tl.getInput("versionSpec", true);
+        let command: string = tl.getInput("command", true);
+        let args: string = tl.getInput("arguments", false);
+        let nuGetPath: string = await nuGetGetter.getNuGet(versionSpec);
+        nutil.setConsoleCodePage();
```

Comment:
it doesn't influence how node writes to stdout. it influences how downstream tools write to stdout (nuget.exe maybe?)